src/test/java/org/springframework/retry/backoff/BackOffPolicyBuilderTests.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.retry.backoff;
+
+import org.junit.Test;
+
+import org.springframework.beans.DirectFieldAccessor;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+/**
+ * @author Tomaz Fernandes
+ * @since 1.3.3
+ */
+public class BackOffPolicyBuilderTests {
+
+	@Test
+	public void shouldCreateDefaultBackOffPolicy() {
+		Sleeper mockSleeper = mock(Sleeper.class);
+		BackOffPolicy backOffPolicy = BackOffPolicyBuilder.newBuilder().withSleeper(mockSleeper).build();
```

Comment:
So, looks like it is just possible to have it like this `BackOffPolicyBuilder.newBuilder().build()`?

Why do we need then this `Sleeper` over here?

Can we have then something like `BackOffPolicyBuilder.newDefaultInstance()` for convenience?

But  yeah: we can live so far with what you did already.

Reply:
> So, looks like it is just possible to have it like this `BackOffPolicyBuilder.newBuilder().build()`?

Yes

> Why do we need then this `Sleeper` over here?

I added the sleeper to all tests because the sleeper addition logic is repeated for all policies. So, doesn't hurt to make sure it's working in all cases, including the default one. I can remove it, specially if we're creating a dedicated method for the default policy.

> Can we have then something like `BackOffPolicyBuilder.newDefaultInstance()` for convenience?

Sure. I think maybe `newDefaultInstance` might give the impression that it's a new default `Builder` instance. Maybe `newDefaultPolicy` instead?

> But yeah: we can live so far with what you did already.

That's nice, thanks.

Reply:
Sure! The `newDefaultPolicy()` sounds better.

Thanks